modules/kmm-symbolic-links-for-in-tree-dependencies.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-symbolic-links-for-in-tree-dependencies_{context}"]
+
+= Symbolic links for in-tree dependencies
+
+Some kernel modules depend on other kernel modules that are shipped with the node's operating system.
+To avoid copying those dependencies into the kmod image, KMM mounts `/usr/lib/modules` into both the build and the
+worker pod's filesystems.
+
+By creating a symlink from `/opt/usr/lib/modules/[kernel-version]/[symlink-name]` to
```

Comment:
Conflicted as to whether these count as user-replaced values. Based on https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values, I think that they are (User-replaced values are often found in places such as code blocks, file paths, and commands.). If we're in agreement that they are, they should be written as follows:

```suggestion
By creating a symlink from `/opt/usr/lib/modules/<kernel_version>/<symlink_name>` to
```

Reply:
Changed. FYI - The bracket style used is the nomenclature for the mid- and upstream versions. But, these are indeed user-replaced values. I'll make all changes.